src/main/resources/assets/create/lang/es_es.json

```diff
@@ -60,15 +60,15 @@
 	"block.create.copper_block": "Bloque de cobre",
 	"block.create.copper_casing": "Bloque de caliza",
 	"block.create.copper_ore": "Mineral de cobre",
-	"block.create.copper_shingles": "Bloque de baldosa de cobre",
-	"block.create.copper_tiles": "Bloque de tejas de cobre",
+	"block.create.copper_shingles": "Teja de cobre",
+	"block.create.copper_tiles": "Baldosa de cobre",
 	"block.create.copper_valve_handle": "Asa de válvula de cobre",
 	"block.create.creative_crate": "Caja creativa",
-	"block.create.creative_fluid_tank": "Tanque de fluídos creativo",
+	"block.create.creative_fluid_tank": "Tanque de fluidos creativo",
 	"block.create.creative_motor": "Motor creativo",
 	"block.create.crushing_wheel": "Rueda trituradora",
 	"block.create.crushing_wheel_controller": "Controlador de Rueda trituradora",
-	"block.create.cuckoo_clock": "Reloj Cucú",
+	"block.create.cuckoo_clock": "Reloj de cuco",
```

Comment:
Cuckoo for its phonetics, it makes cuckoo and not cuckoo (?)

Reply:
Both are used, but it's more common reloj de cuco, cucú comes from fonethics you are right

Reply:
I just checked and actually "Reloj de cuco" is accepted by RAE